src/config.ts

```diff
@@ -110,6 +111,11 @@ export function parseConfig(): DeploymentsConfig | DeploymentStackConfig {
   const description = getOptionalStringInput("description");
   const tags = getOptionalStringDictionaryInput("tags");
   const maskedOutputs = getOptionalStringArrayInput("masked-outputs");
+  const cloud = getOptionalEnumInput("cloud", [
+    "azureChinaCloud",
+    "azureUSGovernment",
+    "azureGermanCloud",
+  ]);
```

Comment:
If we want to add "azureCloud", can default to this, rather than using "null" to represent default.

```suggestion
  const cloud = getOptionalEnumInput("cloud", [
    "azureChinaCloud",
    "azureUSGovernment",
    "azureGermanCloud",
  ]) ?? "azureCloud";
```